db/migrate/20150203151605_cleanup_old_fields.rb

```diff
@@ -22,9 +22,17 @@ def change
     remove_column :courses, :late_penalty_old, :float
     remove_column :courses, :version_penalty_old, :float
 
-    remove_column :submissions, :tweak_old, :float
-    remove_column :submissions, :ignored_old, :boolean
-    remove_column :submissions, :absolute_tweak, :boolean
+    if column_exists? :submissions, :tweak_old
+        remove_column :submissions, :tweak_old, :float
+    end
+    
+    if column_exists? :submissions, :ignored_old
+        remove_column :submissions, :ignored_old, :boolean
+    end
+
+    if column_exists? :submissions, :absolute_tweak  
+        remove_column :submissions, :absolute_tweak, :boolean
+    end
```

Comment:
I just want to be very sure that these migrations are going to run and that the schema file will be accurate.